libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java

```diff
@@ -146,6 +146,14 @@ public NavigationMapRoute(@Nullable MapboxNavigation navigation, @NonNull MapVie
     this.mapRouteProgressChangeListener = new MapRouteProgressChangeListener(this.routeLine, routeArrow);
     initializeDidFinishLoadingStyleListener();
     addListeners();
```

Comment:
nit: This is unnecessary, it will be called when the started state is reached anyway.